cumulus/tasks/discover-granules/package.json

```diff
@@ -13,8 +13,8 @@
   "scripts": {
     "test": "ava",
     "local": "node index.js | pino",
-    "build": "webpack --progress",
-    "watch": "webpack --progress -w",
+    "build": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack --progress",
+    "watch": "rm -rf dist && mkdir dist && cp -R schemas dist/ && webpack --progress -w",
```

Comment:
Does this actually need to be in the watch step?  It only really matters for creating the zip file that's deployed, and that relies on the build step (if I'm correct).

Reply:
I usually try to keep the behavior of `build` and `watch` really similar. In this case including it as part of `watch` will be useful for working on tasks when they are `npm link`ed into deployments, which is something I've needed to do before.